Windows/Python/Install.ps1

```diff
@@ -47,108 +46,114 @@ $anacondaPath2 = "C:\ProgramData\Anaconda3"
 if ((Test-Path $minicondaPath1) -or (Test-Path $minicondaPath2) -or (Test-Path $anacondaPath1) -or (Test-Path $anacondaPath2)) {
     Write-Output "Miniconda or Anaconda is already installed. Skipping Miniconda installation."
     Write-Output "If you wish to install Miniconda using this script, please uninstall the existing Anaconda/Miniconda installation and run the script again."
+    Write-Output ""
+    Write-Output "Please visit the following for more information: "
+    Write-Output ""
+    Write-Output "    https://pythonsupport.dtu.dk/uninstall/conda.html"
+    Write-Output ""
+    Write-Output "IMPORTANT NOTE!!!"
+    Write-Output "The python installation aborted because you already have anaconda/miniconda installed. You are still missing important steps in the installation. Please uninstall Anaconda and run the script again"
+    Write-Output ""
+    exit 1
+}
+
+
+# Script by Python Installation Support DTU
+Write-Output "This script will install Python along with Visual Studio Code - and everything you need to get started with programming"
```

Comment:
`get started with the 1st semester at DTU` (not programming specific)

Reply:
895a874